database.class.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+    include_once 'databaseManager.class.php';
+
+    /**
+     * Database connection
+     */
+    class Database extends DatabaseManager
```

Comment:
@lumakos Is this a database driver for a specific Database Server or just a PDO wrapper?

Reply:
It's a database driver for a specific Database Server. I should rename it to something more specific e.x. class mysqlDatabase extends DatabaseManager.